packages/core-mobile/app/vmModule/ApprovalController/handleAvalancheSendTransaction.ts

```diff
@@ -0,0 +1,81 @@
+import { rpcErrors } from '@metamask/rpc-errors'
+import { Account } from 'store/account/types'
+import walletService from 'services/wallet/WalletService'
+import networkService from 'services/network/NetworkService'
+import { ApprovalResponse, Hex, RpcRequest } from '@avalabs/vm-module-types'
+import { EVM, EVMUnsignedTx, UnsignedTx } from '@avalabs/avalanchejs'
+import { Avalanche } from '@avalabs/core-wallets-sdk'
+
+export const handleAvalancheSendTransaction = async ({
+  request,
+  unsignedTxJson,
+  vm,
+  account,
+  isTestnet,
+  resolve
+}: {
+  request: RpcRequest
+  unsignedTxJson: string
+  vm: 'EVM' | 'AVM' | 'PVM'
+  account: Account
+  isTestnet?: boolean
+  resolve: (value: ApprovalResponse) => void
+}): Promise<void> => {
+  try {
+    // @ts-ignore
+    const externalIndices = request.params?.externalIndices ?? []
+    // @ts-ignore
+    const internalIndices = request.params?.internalIndices ?? []
```

Comment:
let's put these into signing data since it's for signing

Reply:
moved to SigningData